bootstrap/manifest_template.json

```diff
@@ -412,6 +412,7 @@
     "release": {
       "env": {
         "FLYNN_MONGO": "mongodb",
+        "MONGO_DATABASE": "admin",
```

Comment:
Shouldn't be necessary, this is just a convenience when debugging and consistency with the other appliances. I don't think it's likely to be used by users as it is only useful for examining stuff like replica set configuration.